controllers/topic.controller.js

```diff
@@ -55,5 +55,23 @@ const postArticleComment = (req, res, next) => {
       }
     });
 };
+const patchUpdateVotes = (req, res, next) => {
+  const { article_id } = req.params;
+  const { inc_votes } = req.body;
 
-module.exports = { getTopics, getApi, getArticleId, getArticles, getArticleIdComments, postArticleComment };
+  if (typeof inc_votes !== 'number') {
+    return res.status(400).send({ msg: 'Invalid input type' });
+  }
+  updateArticleVotes(article_id, inc_votes)
+  .then((updatedArticle) => {
+    res.status(200).send({ updatedArticle });
+  })
+  .catch((err) => {
+    if (err.message === "Not Found") {
+      res.status(404).send({ msg: "Not Found" });
```

Comment:
Better to put this logic in error handler instead of handling it in the controller